src/Google.Maps.Test/Direction/DirectionRequestTests.cs

```diff
@@ -23,9 +23,9 @@ static DirectionRequestAccessor()
 			{
 				S_instanceType = typeof(DirectionRequest);
 
-				try { _ToUri = S_instanceType.GetMethod("ToUri", BindingFlags.NonPublic | BindingFlags.Instance, null, new Type[] { }, new ParameterModifier[] { }); }
```

Comment:
lol.  reflection api??? ugh!  we should make an interface for ToUri or just make it public.  I forget why I didn't necessarily want it to be a public method, but that was a long time ago.